tools/importer-rest-api-specs/components/transformer/models_to_api.go

```diff
@@ -4,20 +4,15 @@
 package transformer
 
 import (
+	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
 	importerModels "github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
-	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
-	"github.com/hashicorp/pandora/tools/sdk/services"
 )
 
-func ApiResourceFromModelResource(schema importerModels.AzureApiResource) (*services.Resource, error) {
-	return &services.Resource{
-		Operations: resourcemanager.ApiOperationDetails{
-			Operations: schema.Operations,
-		},
-		Schema: resourcemanager.ApiSchemaDetails{
-			Constants:   schema.Constants,
-			Models:      schema.Models,
-			ResourceIds: schema.ResourceIds,
-		},
+func ApiResourceFromModelResource(schema importerModels.AzureApiResource) (*models.APIResource, error) {
```

Comment:
Not a huge deal but I had some initial confusion when looking at the filename which was models **to** api but the function defined within it is api **from** model. Would it make sense to bring those in line one way or another?

Reply:
TBH the entire `transformer` directory will be being removed once the `resourcemanager` models are removed, so I'm going to suggest we leave this as-is for now, since it'll be removed shortly